app/views/state_file/questions/md_permanently_disabled/edit.html.erb

```diff
@@ -36,39 +35,13 @@
         </div>
       </div>
 
-      <div class="question-with-follow-up__follow-up" id="primary-disability-proof">
-        <div class="white-group">
-          <%= f.cfa_radio_set(:primary_proof_of_disability_submitted, label_text: t(".primary_proof_question_html"), collection: [
-            { value: "yes", label: t("general.affirmative") },
-            { value: "no", label: t("general.negative") },
-          ]) %>
-        </div>
-      </div>
-
-      <div class="question-with-follow-up__follow-up" id="spouse-disability-proof">
-        <div class="white-group">
-          <%= f.cfa_radio_set(:spouse_proof_of_disability_submitted, label_text: t(".spouse_proof_question_html"), collection: [
-            { value: "yes", label: t("general.affirmative") },
-            { value: "no", label: t("general.negative") },
-          ]) %>
-        </div>
-      </div>
+      <% if should_warn_about_pension_exclusion %>
+        <%= render partial: "state_file/questions/md_permanently_disabled/primary", locals: { f: f } %>
+      <% end %>
 
-      <div class="question-with-follow-up__follow-up" id="both-disability-proof">
-        <div class="white-group">
-          <%= f.cfa_radio_set(:primary_proof_of_disability_submitted, label_text: t(".primary_proof_question_html"), collection: [
-            { value: "yes", label: t("general.affirmative") },
-            { value: "no", label: t("general.negative") },
-          ]) %>
-        </div>
-
-        <div class="white-group">
-          <%= f.cfa_radio_set(:spouse_proof_of_disability_submitted, label_text: t(".spouse_proof_question_html"), collection: [
-            { value: "yes", label: t("general.affirmative") },
-            { value: "no", label: t("general.negative") },
-          ]) %>
-        </div>
-      </div>
+      <% if should_warn_about_pension_exclusion && current_intake.filing_status_mfj? %>
+        <%= render partial: "state_file/questions/md_permanently_disabled/spouse", locals: { f: f } %>
```

Comment:
i don't know if these need to be partials anymore but ambivalent

Reply:
I like them as partials but I am happy to change it back.

Reply:
Eh just keep them 🤷🏻‍♀️